<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imageCreatorForm</class>
 <widget class="QWidget" name="imageCreatorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image creator</string>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="vShipLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="shipLengthLabel">
         <property name="text">
          <string>Ship length:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="shipLengthBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="resolutionLabel">
         <property name="text">
          <string>Resolution:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="resolutionBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="folderLayout">
     <item>
      <widget class="QLabel" name="folderLabel">
       <property name="text">
        <string>Folder:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectFolder">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QHBoxLayout" name="modelLayout">
     <item>
      <widget class="QCheckBox" name="modelBox">
       <property name="text">
        <string>Model is load</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadModel">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <layout class="QVBoxLayout" name="vObLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="minAngleXZ">
         <property name="text">
          <string>Min obliquity angle:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="minXZBox">
         <property name="minimum">
          <number>20</number>
         </property>
         <property name="maximum">
          <number>79</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="maxAngleXZ">
         <property name="text">
          <string>Max obliquity angle:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="maxXZBox">
         <property name="minimum">
          <number>21</number>
         </property>
         <property name="maximum">
          <number>80</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="angleStepXZ">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Obliquity Angle step:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="angleStepXZBox">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="3">
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonsLowLayout">
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="vAzLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="minAngleXY">
         <property name="text">
          <string>Min angle azimuth:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="minXYBox">
         <property name="maximum">
          <number>359</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="maxAngleXY">
         <property name="text">
          <string>Max angle azimuth:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="maxXYBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="angleStepXY">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Azimuth Angle step:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="angleStepBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="cbTests">
     <property name="text">
      <string>For tests</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
